README.md

```diff
@@ -21,94 +21,3 @@ Fill me in please! Don’t forget code examples:
 ```
 
```

Comment:
Wound't it be easier to rename README.md first and commit and then, add nbdev changes?
In this way everything would be cleaner to see.